docs/api/apiv3/tags/oauth.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: |-
+  TBD
+name: OAuth 2
```

Comment:
The file is not referenced in the `tags` section of `openapi-spec.yml`

Reply:
uf, eagle eye ;)

added